scripts/_fm-creator.js

```diff
@@ -0,0 +1,33 @@
+const fs = require('fs');
+const path = require('path');
+
+const postFolder = '../_posts';
+
+// Frontmatter Creator
+fs.readdir(postFolder, (err, files) => {
+  files.forEach(f => {
+    if (f === path.basename(__filename)) return;
+
+    fs.readFile(path.join(postFolder, f), 'utf-8', (_, data) => {
```

Comment:
Never ignore errors on purpose:

```suggestion
    fs.readFile(path.join(postFolder, f), 'utf-8', (error, data) => {
        if (error) throw error;
```